README.md

```diff
@@ -1,5 +1,68 @@
 # parametric-plasma-source
-Source and build files for parametric plasma source for use in fusion neutron transport calculations.
+
+Python package, C++ source and build files for parametric plasma source for use in fusion neutron transport calculations with OpenMC.
 
 The plasma source is based on a paper by [C. Fausser et al](https://www.sciencedirect.com/science/article/pii/S0920379612000853)
 
+# Installation
+
+```pip install parametric_plasma_source```
+
+# Usage
+
+The parametric plasma source can be imported an used in Python 3 in the following manner.
+
+```
+from parametric_plasma_source import Plasma
+my_plasma = Plasma(major_radius=6,
+                   minor_radius=1.5,
+                   elongation = 2.0
```

Comment:
```suggestion
                   elongation = 2.0,
```

Reply:
Thanks @Shimwell - I've fixed the typo and pushed to this PR.